lexicons/app/bsky/feed/post.json

```diff
@@ -9,11 +9,16 @@
         "type": "object",
         "required": ["text", "createdAt"],
         "properties": {
-          "text": {"type": "string", "maxLength": 256},
+          "text": {"type": "string", "maxLength": 3000, "maxGraphemes": 300},
           "entities": {
             "type": "array",
+            "description": "Deprecated: replaced by app.bsky.richtext.facet.",
             "items": {"type": "ref", "ref": "#entity"}
           },
+          "facets": {
+            "type": "array",
+            "items": {"type": "ref", "ref": "app.bsky.richtext.facet"}
+          },
           "reply": {"type": "ref", "ref": "#replyRef"},
           "embed": {
```

Comment:
@devinivy Are we going to add `images`? (If so make sure to update the postsView in defs.)

Reply:
Yep, I'll plan to hit that today.